docs/concepts/07_dip/05_dapp_developer.md

```diff
@@ -26,8 +26,128 @@ Import the SDK into your code:
+A base proof is a cross-chain state proof, revealing the parts of a DID stored on the KILT blockchain.
+:::
+
+The configuration for the base proof takes the following parameters:
+
+-   `DipSiblingBaseProofInput`: The input for the base proof. It can be one of the following values:
+    -   `didUri`: (Required) The DID URI of the DIP subject performing the cross-chain operation.
+    -   `keyIds`: (Required) An array of verification method IDs of the DID revealed in the cross-chain operation.
+    -   `proofVersion`: (Required) The version of the DIP proof to generate.
+    -   `blockNumber`: The block number of the relay chain to use for the generation of the DIP proof. If not provided, uses the last finalized block.
+    -   `linkedAccounts`: An array of account addresses to link to the DID and reveal in the generated proof.
+    -   `web3Name`: Whether to reveal the web3name of the DID subject in the generated proof.
+
+In the example, the configuration also has extra parameters for the time-bound DID signature extension [mentioned below](#creating-extensions-for-specific-proofs).
+
+For this example, the configuration also needs details of the provider, which in this example case uses this value populated from an environment variable:
+
+```typescript
+const providerAddress = `ws://127.0.0.1:${process.env['PROVIDER_ALICE_RPC']}`
+```
+
+### 2. Generate a submittable extrinsic
+
+The method returns the DID base proof. You have to call a second method, the `[generateDipSubmittableExtrinsic](https://kiltprotocol.github.io/dip-sdk/functions/generateDipSubmittableExtrinsic.html)` method to generate a submittable extrinsic.
+
+You need to pass the following parameters:
+
+-   The API of the consumer chain.
+-   The base proof.
+-   The extrinsic call to the consumer chain.
+-   The DID URI.
+
+:::info Submittable extrinsics
+
+A transaction that originates from an external account and affects the state of the blockchain.
+An extrinisc executes actions on the network, such as transferring funds, making governance decisions, using functionality of the parachain, or interacting with smart contracts.
+
+:::
+
+```typescript
+const dipSubmittable = DipSdk.generateDipSubmittableExtrinsic({
+    api: consumerApi,
+    baseDipProof,
+    call,
+    didUri: did.uri,
+})
+```
+
+The method returns the different components of the proof, [which you can see in the example code](https://github.com/KILTprotocol/dip-sdk/blob/9ad141b3757e076744ab8b2d29bcf10bbeaddd9f/tests/dip-provider-template-dip-consumer-template/develop.test.ts#L219):
+
+-   The provider head proof, which is proof of the provider parachain header on the relay chain.
+-   The commitment proof, which proves the deep commitment for a specific subject, which is the DID URI.
+-   The actual deep proof, which reveals parts of the DID document as specified by key IDs, proof version, whether to include the web rename and the linked account, and the proof version that the user specified.
```

Comment:
```suggestion
-   The DID proof, which reveals parts of the DID document as specified by key IDs, proof version, and whether to include the web3name and any of its linked accounts.
```

I would not use "actual", since a DIP proof is a combination of the three different "sub-proofs". In this case I woudl call it DID proof.